### PR TITLE
Bugfix: unexpected global package import

### PR DESCRIPTION
**Pull request by da-h**
# Bugfix: Unreliable imports.

This MR finalizes #112 by fixing an unexpected global package-import instead of a local minifflask module.

## Description

From #112 on, miniflask tries to use pythons `import` statement if possible to load modules.
The solution was to import the module name using the `importname`, i.e. without the module-base-directory in its identifier. Interestingly, this indeed loaded the correct modules in most cases.
However, in case a system-module with that name exists, miniflask would load that instead of the local module.

**Setup**:
1. Install any python package under the name `setup`.
2. Run `pytest`.

**Problem**:
Due to a module under `./tests/event/beforeafter/modules` named `setup`, the setup above will result in an error.

## Things done in this MR
The solution is quite simple: the module-`id` actually contains the base-directory name (in case the module directory has not been named otherwise in the `mf.init`-call - thus the string is correct in exactly those cases in which we want the internal `import` to be used.

There is one special case, that needed to be resolved to make this work consistently:
The miniflask-internal modules `settings`, `events`, `modules`, etc. have been placed in `miniflask/module/*`, but have been referenced using `miniflask.*` instead of `miniflask.module.*`. To make things consistent, this MR moves all those modules directly into the miniflask base directory.

**Check all before creating this PR**:
- [ ] Documentation adapted
- [ ] unit tests adapted / created

### CODE REVIEW

**Comment by da-h** [comment]
Just a side-note: The linting errors (solved in #113) will throw errors here.
I will rebase whenever #113 is merged.

File: src/miniflask/miniflask.py [hunk 1]
```diff
@@ -177,10 +188,10 @@ def import_module(self, module_name):
 
         # system import (e.g. miniflask internal modules or pip-module repositories)
         if importpath == "system":
-            return import_module(module_spec["importname"])
+            return import_module(module_spec["id"])
```
Comment (sbrodehl) [inline src/miniflask/miniflask.py:191]: Is there special care needed in case the module directory has been renamed in the mf.init-call?

**Review by sbrodehl — CHANGES_REQUESTED** [review]
Looks good so far, just two comments in the code on possibly duplicated code and regarding the renaming of modules in the mf.init-call.

File: src/miniflask/__init__.py [hunk 2]
```diff
@@ -16,3 +16,15 @@
...
+def registerPredefined(modules_avail):
+    for m in ["modules", "events", "info", "settings", "definitions"]:
+        module_name_id = 'miniflask.' + m
+        importname = 'miniflask.modules.' + m
+        modules_avail[module_name_id] = {
+            'id': module_name_id,
+            'importpath': "system",
+            'importname': importname,
+            'lowpriority': False
+        }
```
[the remark this answers is not in the document]
Comment (da-h) [reply src/miniflask/__init__.py:30]: That's indeed a mistake. Thanks for pointing this out!

File: src/miniflask/miniflask.py (hunk 1 above)
Comment (sbrodehl) [reply src/miniflask/miniflask.py:191]: Thanks for clarifying! That would be an interesting way foreward, let's discuss this further in another PR/Issue.

**Comment by da-h** [comment]
Rebased onto master.

**Review by sbrodehl — APPROVED** [review]

**Pull request merged**